pkg/oc/cli/cmd/newapp.go

```diff
@@ -385,13 +390,45 @@ func (o *NewAppOptions) RunNewApp() error {
 			fmt.Fprintf(out, "%sTrack installation of %s with '%s logs %s'.\n", indent, installing[i].Name, o.BaseName, installing[i].Name)
 		}
 	case len(result.List.Items) > 0:
+		//if we dont find a route we give a message to expose it
+		if !containsRoute(result.List.Items) {
+			//we if dont have any route but have services - suggest commndas to expose those
+			svc := getAllServices(result.List.Items)
+			if len(svc) > 0 {
+				fmt.Fprintf(out, "%sApplication is not exposed. You can expose it to outside world by executing one or all of the commands below \n", indent)
```

Comment:
Not sure on the final wording for this: maybe the following is better?
`"%sApp is not exposed. You can expose services to the outside world by executing one or more of the commands below:\n"`